piccolo_api/media/s3.py

```diff
@@ -38,12 +42,22 @@ def __init__(
             The Piccolo ``Column`` which the storage is for.
         :param bucket_name:
             Which S3 bucket the files are stored in.
-        :param folder:
+        :param folder_name:
             The files will be stored in this folder within the bucket. S3
             buckets don't really have folders, but if ``folder`` is
             ``'movie_screenshots'``, then we store the file at
             ``'movie_screenshots/my-file-abc-123.jpeg'``, to simulate it being
             in a folder.
+        :param cache_max_age
+            It takes the value in second
+            For example::
+                cache_max_age=86400
+                This will keep the cache for 24 hour.
+        :param default_acl
```

Comment:
And here

Reply:
Corrected, Please do merge and release.

Reply:
@sumitsharansatsangi Can you run `./scripts/lint.sh` because you have 3 long lines (line E501 too long) and 1 Mypy error (incompatible types in  assignment). If you don't fix that linter will complain and linter in GH actions won't pass.

Reply:
Ok, let me do.

Reply:
Corrected all the linter error,